neurow/lib/neurow/observability/message_broker_stats.ex

```diff
@@ -0,0 +1,82 @@
+defmodule Neurow.Observability.MessageBrokerStats do
+  use Prometheus.Metric
+
+  def setup() do
+    Gauge.declare(
```

Comment:
**Suggestion** 🤔 : should we group the 3 "use case" in dedicated methods ?

- declare
- init
- monitor